bin/packages/build.js

```diff
@@ -36,6 +36,8 @@ const DONE = chalk.reset.inverse.bold.green( ' DONE ' );
  */
 const babelDefaultConfig = require( '@wordpress/babel-preset-default' );
 babelDefaultConfig.babelrc = false;
+// TODO: It should become the default value when all modules are moved to packages.
+babelDefaultConfig.plugins[ 1 ][ 1 ].pragma = 'createElement';
```

Comment:
1. Must we mutate? Can we assign into a new object? Not sure what would break, but certainly seems prone to cause breakage as implemented.
2. Similarly, the `[ 1 ][ 1 ]` assumes a very specific order which, while we maintain and is unlikely to change anytime soon, is certainly incredibly fragile.

Reply:
I opted for a quick fix with the hope that it won't last for long. I guess it is going to be around for a couple of weeks at least so I'm happy to provide more solid processing of default Babel config.

Reply:
https://github.com/WordPress/gutenberg/pull/6828/commits/fdb107e024dc6e363c892f43b89a6faec3d8dbe2 - done